README.md

```diff
@@ -68,9 +68,9 @@ Running
 -------
 Run the following commands.
 ```
-bash bash scripts/run.sh <robot-name>
+bash scripts/run.sh <robot-name>
 ```
-<robot-name> could be `han` or `sophia`
+<robot-name> could be `han` or `sophia10`
```

Comment:
last time I looked, sophia10 was in some private repo. Can you pull the needed strings to get this moved to a public repo?  Ditto for the arms?

Reply:
@elggem  see Linas's suggestion